diff -urN cdrkit-1.1.0.org/readom/readom.c cdrkit-1.1.0/readom/readom.c
--- cdrkit-1.1.0.org/readom/readom.c	2006-11-25 13:13:05.000000000 +0100
+++ cdrkit-1.1.0/readom/readom.c	2006-12-24 14:42:40.421054000 +0100
@@ -315,6 +315,8 @@
 	if (help)
 		usage(0);
   if (pversion) {
+     printf("readcd %s is not what you see here. This line is only a fake for too clever\n"
+		     "GUIs and other frontend applications. In fact, this program is:\n", cdr_version);
      printf("readom " CDRKIT_VERSION " (" HOST_SYSTEM ")\n"
            "Copyright (C) 1987, 1995-2006 Joerg Schilling\n"
            "Copyright (C) 2006 Cdrkit maintainers\n"
