--- cdrkit-1.1.11/genisoimage/genisoimage.c.formatsec	2013-02-14 21:43:12.811173895 +0100
+++ cdrkit-1.1.11/genisoimage/genisoimage.c	2013-02-14 21:46:32.356759530 +0100
@@ -3406,7 +3406,7 @@ if (check_session == 0)
 	if (goof) {
 		fprintf(stderr, "ISO9660/Rock Ridge tree sort failed.\n");
 		if(merge_warn_msg)
-			fprintf(stderr, merge_warn_msg);
+			fputs(merge_warn_msg, stderr);
 		exit(1);
 	}
 #ifdef UDF
@@ -3419,7 +3419,7 @@ if (check_session == 0)
 	if (goof) {
 		fprintf(stderr, "Joliet tree sort failed. The -joliet-long switch may help you.\n");
 		if(merge_warn_msg)
-			fprintf(stderr, merge_warn_msg);
+			fputs(merge_warn_msg, stderr);
 		exit(1);
 	}
 	/*
--- cdrkit-1.1.11/genisoimage/tree.c.formatsec	2013-02-14 21:48:17.412857059 +0100
+++ cdrkit-1.1.11/genisoimage/tree.c	2013-02-14 21:48:36.219421842 +0100
@@ -647,7 +647,7 @@ got_valid_name:
      fprintf(stderr, "Unable to sort directory %s\n",
            this_dir->whole_name);
      if(merge_warn_msg)
-        fprintf(stderr, merge_warn_msg);
+        fputs(merge_warn_msg, stderr);
      exit(1);
 	}
 	/*
